src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -11,7 +11,9 @@
 
   <ItemGroup>
     <Folder Include="wwwroot\" />
-    <Folder Include="Localization\" />
+    <Content Include="Localization\**" >
```

Comment:
Why does it work differently from the `wwwroot` folder? Is that because the SDK does it automatically?

Reply:
That's what I wanna say

Reply:
I am not sure. All I know is that  <Folder Include="Localization\" /> did not copy the files to the output.

Reply:
Yes it is part of the sdk to publish static assets and excluding some items and folders as `node_modules`. Maybe we could do the same for `Localization` as we do for `App_Data` in `OC.Application.Cms.Core.Targets.targets`. And maybe also to auto create a place holder file if it doesn't exist yet as we do for `wwwroot`.

    <ItemGroup>
      <Compile Remove="App_Data\**" />
      <Content Remove="App_Data\**" />
      <EmbeddedResource Remove="App_Data\**" />
      <None Remove="App_Data\**" />
    </ItemGroup>

    <ItemGroup>
      <ResolvedFileToPublish Include="App_Data\**" Exclude="App_Data\**\*.log">
        <RelativePath>App_Data\%(RecursiveDir)%(Filename)%(Extension)</RelativePath>
      </ResolvedFileToPublish>
    </ItemGroup>

    <Target Name="MakeWebRootFolder" BeforeTargets="Build" Condition="!Exists('wwwroot/.placeholder')">
      <WriteLinesToFile Lines="" Encoding="Unicode" File="wwwroot/.placeholder" />
    </Target>